test_stack.py

```diff
@@ -191,3 +196,70 @@ def test_dynamic_jump():
         eval_program("push 2; jump")
     with pytest.raises(IndexError):
         eval_program("push -2; jump; jump 0")
+
+
+@pytest.mark.timeout(1)
+def test_jump_labels():
+    # A jump label should be converted into relative jumps at parse time.
+    expected = [Instr('push', [1]), Instr('push', [2]), Instr('to', [1])]
+    program = "@start\npush 1; push 2; jump @start"
+    assert list(stack.parse_program(program)) == expected
+
+    # Multiple jump labels should work.
+
+@pytest.mark.timeout(1)
+def test_fibonacci():
+    # Has the nth fibonacci term at the top of the stack
+    # TODO: test against different fibonacci inputs.
+    # (0 = 0th term)
+    program = """
+    push 1
+    push 0
+    push 6
+    @setup
+    swap 2
+    @next_fib
+    dup
+    swap 2
+    add
+    @decement_n
+    swap 2
+    push -1
+    add
+    @check_for_zero
+    quiet not
+    push 1
+    add
+    jump
+    jump @setup
+    jump @end
+    @end
+    swap 2
+    pop
+    swap
+    pop"""
+    assert eval_program(program) == [8]
+
+@pytest.mark.timeout(1)
+def test_absolute_jump():
+    # An absolute jump will jump to the instruction number
+    # reguardless of where the jump is placed. Instructions start at 0.
+    eval_program('to 3; push 2; push 3; push 4') == [3, 4]
```

Comment:
You have these instructions starting at 1, instead of 0, it looks like. I think your tests are wrong, and the implementation is wrong in a way that matches the tests.